composer.json

```
@@ -16,5 +16,8 @@
     "require-dev": {
         "jakub-onderka/php-parallel-lint": "^0.9.2",
         "jakub-onderka/php-console-highlighter": "^0.3.2"
-    }
+    },
+  "scripts": {
+    "lint": "find . -name \\*.php -not -path './lib/composer/*' -exec php -l \"{}\" \\;"
```

```suggestion
    "lint": "parallel-lint --exclude lib/composer/jakub-onderka/ --exclude 3rdparty/symfony/polyfill-php70/Resources/stubs/ --exclude 3rdparty/patchwork/utf8/src/Patchwork/Utf8/Bootup/ --exclude 3rdparty/paragonie/random_compat/lib/ --exclude lib/composer/composer/autoload_static.php --exclude 3rdparty/composer/autoload_static.php --exclude 3rdparty/doctrine/cache/lib/Doctrine/Common/Cache/RiakCache.php ."
```

(from the Drone script)

I guess we can just ignore 3rdparty all together

---

yeah it is not checked out right now :P
Nor do we run composer install.
I mean it is done now within a minute so I'm not sure we need to invoke composer install and then parallel lint...

---

Fine by me then :)